rskj-core/src/main/java/org/ethereum/config/blockchain/upgrades/ConsensusRule.java

```diff
@@ -54,6 +54,7 @@
     RSKIP171("rskip171"),
     RSKIP172("rskip172"),
     RSKIP174("rskip174"),
+    RSKIP179("rskip179"),
```

Comment:
Could you leave a comment here, telling what #179 is all about, short comment (title?)?
We are trying to identify faster whats the main reason behind the code.

Reply:
you can find some details of this RSKIP [here](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP179.md)

Reply:
done